### PR TITLE
Add tasks for exposing endpoints

### PR DESCRIPTION
**Pull request by yrobla**
Run haproxy on the disconnecter vm, to expose
the cloud internal endpoints.

Signed-off-by: Yolanda Robla <yroblamo@redhat.com>

### CODE REVIEW

**Review by vitus133 — APPROVED** [review]

File: ztp/ztp-playbooks/install-ai-operator/roles/expose-endpoints/templates/create_haproxy.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,103 @@
+#! /bin/bash
+
+# install haproxy if not done
+dnf -y install haproxy
+
+mkdir -p /etc/haproxy
+chmod 0755 /etc/haproxy
+
+rm /etc/haproxy/haproxy.cfg
```
Comment (serngawy) [inline ztp/ztp-playbooks/install-ai-operator/roles/expose-endpoints/templates/create_haproxy.sh.j2:9]: This will fail, you just create the dir before it.

**Pull request merged**